python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
```

Comment:
This could be defined outside of the function since it is used twice. We could also ask the user to specify if they want "y" to be counted as a vowel since it is ambiguous.